extensions/wikia/Discussions/maintenance/ForumDumper.php

```diff
@@ -0,0 +1,246 @@
+<?php
+
+namespace Discussions;
+
+class ForumDumper {
+
+	const FORUM_NAMESPACES = [ 2000, 2001 ];
+
+	const TABLE_PAGE = 'page';
+	const TABLE_COMMENTS = 'comments_index';
+	const TABLE_REVISION = 'revision';
+	const TABLE_TEXT = 'text';
+	const TABLE_VOTE = 'page_vote';
+
+	const CONTRIBUTOR_TYPE_USER = "user";
+	const CONTRIBUTOR_TYPE_IP = "ip";
+	const CONTRIBUTOR_TYPE_DELETED = "[deleted]";
+	const CONTRIBUTOR_TYPE_UNKNOWN = "[unknown]";
+
+	const REGEXP_MATCH_TITLE = '/<ac_metadata.*title="([^"]*)".*>.*<\/ac_metadata>/';
+
+	const COLUMNS_PAGE = [
+		"page_id",
+		"namespace",
+		"raw_title",
+		"is_redirect",
+		"is_new",
+		"touched",
+		"latest_revision_id",
+		"length",
+		"parent_page_id",
+		"parent_comment_id",
+		"last_child_comment_id",
+		"archived_ind",
+		"deleted_ind",
+		"removed_ind",
+		"locked_ind",
+		"protected_ind",
+		"sticky_ind",
+		"first_revision_id",
+		"last_revision_id",
+		"comment_timestamp"
+	];
+
+	const COLUMNS_REVISION = [
+		"revision_id",
+		"page_id",
+		"page_namespace",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "page_namespace" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)